Tools/setup/ubuntu.sh

```diff
@@ -28,19 +30,31 @@ do
 		INSTALL_SIM="false"
 	fi
 
-done
+	if [[ $arg == "--with-rtps" ]]; then
+		INSTALL_RTPS="true"
+	fi
 
-# detect if running in docker
-if [ -f /.dockerenv ]; then
-	echo "Running within docker, installing initial dependencies";
-	apt-get --quiet -y update && DEBIAN_FRONTEND=noninteractive apt-get --quiet -y install \
-		ca-certificates \
-		gnupg \
-		lsb-core \
-		sudo \
-		wget \
-		;
-fi
+	if [[ $arg == "--with-java" ]]; then
```

Comment:
Why specify? It's either a required dependency or not, can't we check?

Reply:
you are right, I removed the argument, but kept the logic that checks if you need to install it